src/module/addremove.js

```diff
@@ -0,0 +1,88 @@
+const add = () => {
+  const addTask = document.getElementById('addTask'); // get the input field element
+  if (addTask.value !== '') {
+    const tasksArray = JSON.parse(localStorage.getItem('tasksArray')) || [];
+    tasksArray.push({ completed: false, description: addTask.value });
+    for (let i = 1; i <= tasksArray.length; i += 1) {
+      tasksArray[i - 1].index = i;
+    }
+    localStorage.setItem('tasksArray', JSON.stringify(tasksArray));
+  }
+  addTask.value = '';
+};
+
+const remove = (index) => {
+  const tasksArray = JSON.parse(localStorage.getItem('tasksArray')) || [];
+  tasksArray.splice(index, 1);
+  for (let i = 1; i <= tasksArray.length; i += 1) {
+    tasksArray[i - 1].index = i;
+  }
+  localStorage.setItem('tasksArray', JSON.stringify(tasksArray));
+};
```

Comment:
- [ ] Good job adding the `add` function and the `edit` task function. However removing  a selected task does not work. When the user tries to delete a random task, the task which gets deleted is the task which was added last.  This is because in line `18`, whenever the remove function is fired, the `tasksArray` is always subtracted by `1` and not by the index of the selected task. Kindly fix this


- [ ] Check the video below for more information.
[Screencast from 16-06-2023 15:55:44.webm](https://github.com/mohashyne/Todo_list/assets/81932478/1dbb3376-6f48-4df3-bb87-e8e52bd85304)

Reply:
Required changes fixed as requested